app/services/db/sp_cost/add_sp_cost.rb

```diff
@@ -18,7 +20,10 @@ class AddSpCost
 
       def self.call(issuer, ial, token)
         return if issuer.nil? || token.blank?
-        return unless TOKEN_WHITELIST.index(token.to_sym)
+        unless TOKEN_WHITELIST.index(token.to_sym)
```

Comment:
```suggestion
        unless TOKEN_WHITELIST.include?(token.to_sym)
```

ditto for `include?`